eelbrain/plot/tests/test_glassbrain.py

```diff
@@ -20,6 +20,30 @@ def test_glassbrain():
     p = plot.GlassBrain(ndvar_30, draw_arrows=False)
     p.close()
 
+    # vmin, vmax, and symmetric colorbar
+    p = plot.GlassBrain(ndvar_30, colorbar = True, cmap = 'lux-a')
```

Comment:
```suggestion
    p = plot.GlassBrain(ndvar_30, colorbar=True, cmap='lux-a')
```
Small stylistic change, also below (https://peps.python.org/pep-0008/#other-recommendations)